include/dpp/discordvoiceclient.h

```diff
@@ -402,6 +547,14 @@ class DPP_EXPORT discord_voice_client : public websocket_client
 	 */
 	uint8_t encode_buffer[65536];
 
+	/**
+	 * @brief DAVE - Discord Audio Visual Encryption
+	 * Used for E2EE encryption. dave_protocol_none is
+	 * the default right now.
+	 * @warning DAVE E2EE is an EXPERIMENTAL feature!
+	 */
+	dave_version_t dave_version;
```

Comment:
Then let's default it to `dave_protocol_none`

Reply:
it is defaulted to this in the constructor, where we initialise everything else, there isnt really anything we initialise in place in this class as far as im aware?

The user sets the default via the constructor but the default variable value there is `dave = false` in an optional parameter.